### DISCUSSION TITLE
Confusión Tests Parte 2

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-1#163] [Tarea 2] [answered]
He estado leyendo las discusiones y terminé confundido. ¿Cuáles son los tests correctos para la parte 2? He descargado los tests en dos ocasiones y ahora no estoy seguro de cuál carpeta es la correcta. Cabe mencionar que con una de las carpetas de tests mi código funciona y da el resultado esperado, pero con la otra no. Sin embargo, no estoy seguro si la carpeta que hace que mi código funcione correctamente es la que tiene los tests vigentes. ¿Podrían proporcionarme un enlace a los tests correctos?

**Comment by Borjampm** [comment, marked as answer]
Hola! Los tests correctos son los que están en el drive, al corregirlos se cambiaron estos pero no la ubicación
